File: uri/1st Page/cp with java/src/com/URI1036.java
package com;

import java.util.Scanner;

import static java.lang.Math.sqrt;

public class URI1036 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double a = sc.nextDouble();
        double b = sc.nextDouble();
        double c = sc.nextDouble();
        double rt = (b * b) - (4 * a * c);
        if(a==0 || rt<0){
            System.out.println("Impossivel calcular");
        }
        else{
            double r1 = (-b + sqrt(rt))/(2*a);
            double r2 = (-b - sqrt(rt))/(2*a);
            System.out.printf("R1 = %.5f\n",r1);
            System.out.printf("R2 = %.5f\n",r2);
        }
    }
}

File: uri/1st Page/cp with c/URI 1036.c
#include<stdio.h>
int main()
{
    double a, b, c, calc, r1, r2;
    scanf("%lf %lf %lf", &a, &b, &c);
    if(a>0.0 && ((b*b)-(4*a*c))>=0)
    {
        calc = sqrt((b*b)-(4*a*c));
        r1 = (-b+calc)/(2*a);
        r2 = (-b-calc)/(2*a);
        printf("R1 = %.5lf\n",r1);
        printf("R2 = %.5lf\n",r2);
    }
    else{
        printf("Impossivel calcular\n");
    }
    return 0;
}
